### PR TITLE
added portfolio to profile component

### PR DESCRIPTION
**Pull request by Keelando**

### CODE REVIEW

**Comment by alexnguyenn** [comment]
It looks out of place for me - can you try grabbing an icon from https://react-icons.github.io/react-icons for that?

File: src/pages/AboutUs.tsx [hunk 1]
```diff
@@ -36,6 +37,17 @@ const Profile = (props: ProfileProps): JSX.Element => {
...
                 )}
+                {props.portfolio && (
+                    <Link
+                        href={props.portfolio}
+                        isExternal
+                        _hover={{ textDecoration: "none", color: "inherit" }}
+                    >
+                        <Button variant="outline" size="sm" rightIcon={<BiLinkExternal />}>
+                            Portfolio
+                        </Button>
```
Comment (gwwatkin) [inline src/pages/AboutUs.tsx:48]: I thought you were going to use something like `fa-globe`

**Comment by Keelando** [comment]
I think the open folder fa icon is most self-explanatory. The runner up in my mind is the FaBriefcase

**Review by gwwatkin — APPROVED** [review]
You know what would be really cool though? having the closed folder most of the time and changing it to open folder on hover :heart_eyes:

**Comment by alexnguyenn** [comment]
sorry totally forgot about this. I don't really like the icon there (it looks out of place to me). My suggestions are FaDev, FaAddressCard, FaCode (favorite), FaFolderOpen (no Reg), FaGlobeAmericas, FaLink.

**Pull request merged**